Update static CSVs (remove duplicated IE613 antenna positions)

David-McKenna opened the pull request:
Hi Michiel,

This PR goes alongside #12 to show the differences in the local CSVs after regenerating them from the current state of the ASTRON git. As I noted in #12,

- This removes duplicated IE613 antenna
- The ordering of the core HBA entries phase centre entries has been modified slightly

Cheers,
David

David-McKenna committed:
Fix IE613 antenna locations being duplicated

etrs-phase-centres.csv update as a result of re-running the scripts being provided in a separate PR

David-McKenna commented:
I suspect the build is failing, and these extra antenna were added, as a focus to get tests compiling after our LBA array definition was adjusted by 6cm ( https://git.astron.nl/ro/lofar/-/commit/b34203f6896e2c0da1d04ca029b185e3a9dfa929 )

I'll update the test cases shortly.

David-McKenna committed:
Update test cases to account for shift in IE613 location

brentjens reviewed (changes requested):
Hi David,

thank you very much for this work! Could you please regenerate the file such that coordinates are rounded to the closest mm?
Thank you so much!
- Michiel

David-McKenna committed:
Reduce phase center precision to 1mm

David-McKenna committed:
Update test cases to account for different phase center again

David-McKenna commented:
Hey Michiel,

I've updated #12 to ensure that it will only print the phase centres to millimetre precision, updated the test cases again to account for the slight change, and pushed the resulting changes to the branch here as well. Do you want me to squash all the commits?

Cheers,
David
